plugin/pkg/admission/alwayspullimages/admission.go

```diff
@@ -101,12 +103,49 @@ func (*AlwaysPullImages) Validate(ctx context.Context, attributes admission.Attr
 	return nil
 }
 
+// check if it's update and it doesn't change the images referenced by the pod spec
+func isUpdateWithNoNewImages(attributes admission.Attributes) bool {
```

Comment:
I guess this could be extracted as a utility function and then can be shared in other admissions. :)